config/initializers/seeds.rb

```diff
@@ -1,21 +1,23 @@
-return unless (ForemanInternal.table_exists? rescue(false)) && !Foreman.in_rake? && !Rails.env.test?
+Rails.application.config.to_prepare do
+  next unless (ForemanInternal.table_exists? rescue(false)) && !Foreman.in_rake? && !Rails.env.test?
 
-if ActiveRecord::Base.connection.migration_context.needs_migration?
-  Rails.logger.warn("Migrations pending, skipping seeding. Please run `foreman-rake db:migrate` manually.")
-  return
-end
+  if ActiveRecord::Base.connection.migration_context.needs_migration?
+    Rails.logger.warn("Migrations pending, skipping seeding. Please run `foreman-rake db:migrate` manually.")
+    next
+  end
 
-Foreman::Application.configure do |app|
-  config.after_initialize do
-    seeder = ForemanSeeder.new
+  Foreman::Application.configure do |app|
+    config.after_initialize do
```

Comment:
This block was already in `after_initialize` I do not think we want to wrap that in `to_prepare`.